packages/modeling/src/operations/booleans/martinez/index.js

```diff
@@ -0,0 +1,45 @@
+import * as geom2 from '../../../geometries/geom2/index.js'
+import boolean from './martinez.js'
+import {
+  INTERSECTION,
+  DIFFERENCE,
+  UNION
+} from './operation.js'
+
+const toMartinez = (geometry) => {
+  const outlines = geom2.toOutlines(geometry)
+  outlines.forEach((outline) => {
+    // Martinez expects first point == last point
+    outline.push(outline[0])
+  })
+  return outlines
+}
+
+const fromMartinez = (outlines) => {
+  if (outlines) {
+    return geom2.fromOutlines(outlines.flat())
+  } else {
+    return geom2.create()
+  }
+}
+
+export const union = (subject, clipping) => {
```

Comment:
OK... I don't mind this but just to follow what's happening with geom3 booleans, shouldn't this be called...

- booleans/unionGeom2Sub
- booleans/intersectGeom2Sub
- booleans/subtractGeom2Sub

These three little functions (and conversion utilities) are really part of JSCAD, not the martinez library.

Reply:
I refactored this code now to get rid of these functions